src/amazonsqs/Module.md

```diff
@@ -225,9 +221,7 @@ public function main(string... args) {
         accessKey: "AKIAIOSFODNN7EXAMPLE",
```

Comment:
Is it okay to add this accessKey in the Module.md?

Reply:
Removed